### ISSUE TITLE
Update table to include data source column - always float default to top, mark default with badge (OUIBadge)

### ISSUE THREAD

**Issue by KrooshalUX** [opensearch-project/ux#38]
![image](https://user-images.githubusercontent.com/105819041/194140494-b34fa1e2-968d-4c7f-bd33-eb00aaec684c.png)

**Comment by zhongnansu** [comment]
I believe this is done, right? @kristenTian can you confirm?

**Comment by kristenTian** [comment]
Yes. Kroosh mentioned she will help create separate task to move around tags in future development.
